hspec-discover/integration-test/hspec-discover-integration-tests.cabal

```diff
@@ -142,6 +142,23 @@ test-suite with-warn-missing-export-lists
     , hspec
   default-language: Haskell2010
 
+test-suite with-warn-prepositive-qualified-module
+  build-tool-depends: hspec-discover:hspec-discover
+  type:
+      exitcode-stdio-1.0
+  ghc-options:
+      -Wprepositive-qualified-module -Werror
+  if !impl(ghc >= 8.10)
+    buildable: false
```

Comment:
Apparently somebody decided that booleans in `.cabal` files should be case sensitive now. 

https://github.com/hspec/hspec/runs/8173846384?check_suite_focus=true#step:4:310

So I guess for for compatibility it makes sense to use `False` here.